package.json

```diff
@@ -179,17 +181,20 @@
     "node-fetch": "^3.2.10",
     "np": "^7.6.2",
     "npm-run-all": "^4.1.5",
+    "patch-package": "^6.4.7",
     "postcss-html": "^1.5.0",
     "postcss-import": "^14.1.0",
     "postcss-less": "^6.0.0",
     "postcss-sass": "^0.5.0",
     "postcss-scss": "^4.0.4",
-    "prettier": "2.7.1",
     "remark-cli": "^11.0.0",
     "sugarss": "^4.0.1",
     "typescript": "^4.7.4"
   },
   "engines": {
     "node": "^12.20.0 || ^14.13.1 || >=16.0.0"
+  },
+  "overrides": {
+    "prettier": "^2.7.1"
```

Comment:
This fixed https://github.com/stylelint/stylelint/pull/6282#discussion_r949054860

Reply:
Could you explain why this `overrides.prettier` is necessary?

Reply:
> Could you explain why this `overrides.prettier` is necessary?

There are different prettier versions installed. `^1. 19` vs `^2. 0` which  could have different defaults O suppose.

Reply:
Is this the same problem as https://github.com/changesets/changesets/issues/616?

Reply:
Seems true.

Reply:
What about ignoring `CHANGELOG.md` like [this workaround](https://github.com/PrairieLearn/PrairieLearn/commit/a9b1af0f21d7e55339533d7bf364545f40011b51)? Because it's unnecessary to change dependencies and `package.json` and possible to leave a TODO comment, in addition likely, we won't modify `CHANGELOG.md` manually with the new workflow.

Reply:
Ignoring will result inconsistent list item maybe, but personally I want a single prettier installed to reduce related issues. https://github.com/PrairieLearn/PrairieLearn/commit/a9b1af0f21d7e55339533d7bf364545f40011b51 this workaround Is not good at all IMO.

And also, Is there any downside by using `overrides`?

Reply:
It's challenging to find the right place. 🤔 

I'm not sure about any downsides with `overrides`, but I have a little worry about dependency management; whether `overrides` may cause other problems.

Reply:
I don't see any downside by overriding `prettier`'s version, it should always be consistent, otherwise different prettier may depend on different prettier version which results in unexpected differences like this due to inconsistent defaults.

Reply:
Sounds good. 👍🏼 
When `chagesets` fixes the problem, let's revert this `overrides` change.